homeassistant/components/meteoalarm/binary_sensor.py

```diff
@@ -49,27 +49,27 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         _LOGGER.error("Wrong country digits, or province name")
         return
 
-    add_entities([MeteoAlertBinarySensor(api, name)], True)
+    add_entities([MeteoAlertbinary sensor(api, name)], True)
```

Comment:
... and this. Only change the comments :)

Reply:
Yes, stupid mistake. I have search and replaced it Oops ;)